scipy/stats/_continuous_distns.py

```diff
@@ -5335,6 +5335,103 @@ def _fitstart(self, data, args=None):
 invweibull = invweibull_gen(a=0, name='invweibull')
 
 
+class jf_skew_t_gen(rv_continuous):
+    r"""Jones and Faddy skew-t distribution [1]_.
+
+    %(before_notes)s
+
+    Notes
+    -----
+    The probability density function for `jf_skew_t` is:
+
+    .. math::
+
+        f(t; a, b) = C_{a,b}^{-1}
+                    \left(1+\frac{t}{\left(a+b+t^2\right)^{1/2}}\right)^{a+1/2}
+                    \left(1-\frac{t}{\left(a+b+t^2\right)^{1/2}}\right)^{b+1/2}
```

Comment:
Matches
<img width="542" alt="image" src="https://github.com/scipy/scipy/assets/6570539/ad78394c-7a8e-48e3-988a-66fc84df28a3">

But maybe a bit too closely. I'd reword some of this slightly.

Reply:
Ok, I've changed it to (in both the doc string and the tutorial file):
```
When :math:`a<b`, the distribution is negatively skewed, and when 
:math:`a>b`, the distribution is positively skewed. If :math:`a=b`, then 
we recover the `t` distribution with :math:`2a` degrees of freedom.
```